api/scaffold/__init__.py

```diff
@@ -2202,6 +2304,11 @@ def connect(
                 self.clocks.append(clock)
                 self.__setattr__(f"clock{i}", clock)
 
+        # Declare the swd module
+        if self.version >= parse_version("0.10"):
+            self.swd = swd = SWD(self)
+            self.__setattr__("swd", swd)
```

Comment:
This line calling `__setattr__` is not necessary as the attribute is already set the line before.